Dockerfile.lmes-job

```diff
@@ -162,7 +209,7 @@ COPY . .
 COPY requirements.txt .
 
 # Install the pinned dependencies
-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt "huggingface_hub[hf_xet]"
```

Comment:
If this dependency is to be introduced, can it be on the `pyproject.toml`?
I'm fine with adding to it to the main deps, i.e.

```
dependencies = [
      # ...
      "huggingface-hub[hf_xet]==0.30.2",  # Add this line
      # ...
  ]
```

The `requirements.txt` would need to be regenerated before committing, though (`poetry export -f requirements.txt --output requirements.txt --without-hashes --extras "api ibm_watsonx_ai ifeval math"`)

Reply:
@satyamg1620 if it's ok with you, let's remove it from this PR (keep this about s390x parquet) and make a separate one with hf_xet, so it's easier to test for regressions and/or revert. wdyt? Thanks!